tsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2020",
```

Comment:
Will see if it breaks something in the future. Can lower it anytime...

Reply:
Personally, I have the experience that `es2017` currently seems to be the best choice
Based on https://node.green

Reply:
Yes, makes sense. I've been too hasty...